### ISSUE TITLE
Rogue Masters (1063590)

### ISSUE THREAD

**Issue by somewhatfrog** [ValveSoftware/Proton#7276]
# Compatibility Report
- Name of the game with compatibility issues: Rogue Masters
- Steam AppID of the game: 1063590

## System Information
- GPU: 3060Ti 8GB
- Video driver version: tested on 545.29.02 and 470.223.02 
- Kernel version: 6.6.2-zen-1-zen
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version: Experimental, 8 and 7
## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1063590.log](https://github.com/ValveSoftware/Proton/files/13437872/steam-1063590.log)

## Symptoms
There is three issues which do not happen on windows, same behavior on three different systems (two PC's and one Steam Deck):
1. Regardless of resolution or graphic settings game maxes out VRAM usage. In main menu, lobby and tutorial game runs smooth, once loaded into the actual map FPS drops to 10-15 unless camera is pointed into the sky or floor. Sometimes game causes full system freeze.
2. Videos don't play (they play with proton GE though).
3. Game crashes on attempts to change to fullscreen.

## Reproduction
Run game, start offline lobby, enter any map.
Change presentation mode from the default one to fullscreen.

**Comment by somewhatfrog** [comment]
A small update, I got some data from another person who uses AMD GPU with 12GB of VRAM, for them game runs perfectly fine except their VRAM usage tops at 10.5GB, while my GPU have only 8GB which is about how much it uses in the lobby for them as well.
I don't know if they will drop their log here, but I hope this info will be helpful to narrow it down a little.

**Comment by GrimKriegor** [comment]
It seems the game uses quite a lot of VRAM, I have seen some Windows users experiencing the exact same issue over the weekend, on 8GiB cards.
Whatever small overhead Proton and company add seems to be enough to push some users over the 8GiB edge quicker.
The developers are aware of the issue and hopefully it gets sorted out by release.

**Comment by somewhatfrog** [comment]
> It seems the game uses quite a lot of VRAM, I have seen some Windows users experiencing the exact same issue over the weekend, on 8GiB cards. Whatever small overhead Proton and company add seems to be enough to push some users over the 8GiB edge quicker. The developers are aware of the issue and hopefully it gets sorted out by release.

This is true, however, I noticed to have the same fps issues even when my gpu reported to only utilize 7.6 out of 8GiB. This same instant fps drop goes away the moment camera is pointed away from the map and instantly returns when pointed back at it and doesn't exist at all in lobby and tutorial regardles of VRAM usage. Also on windows, when players get fps drop from VRAM thing, their GPU usage stays quite low, while here it goes to 100% the moment camera is pointed at the map. So there might be something else going as well.
